client/jetpack-connect/plans.jsx

```diff
@@ -271,9 +268,6 @@ class Plans extends Component {
 		return (
 			<div>
 				<QueryPlans />
-				{ this.props.selectedSite ? (
-					<QuerySitePlans siteId={ this.props.selectedSite.ID } />
```

Comment:
We may still need the query for redirecting if the user already has a paid plan?

Reply:
Yeah I think it is needed for [`isSiteOnPaidPlan()`](https://github.com/Automattic/wp-calypso/blob/master/client/state/selectors/is-site-on-paid-plan.js).

Reply:
👍  Nice catch and a good reminder that often these query components don't have a "direct" relationship to props. I'll need to be careful as I _rip it out_.

In this case, the issue is actually hidden by the fact that this query component mounts a few levels deeper on `PlanFeaturesMain`. That usage could use it's own audit, but I'll leave that for another day…

I've rebased and dropped this change.